src/ngraph/runtime/cpu/cpu_external_function.cpp

```diff
@@ -289,6 +289,9 @@ static const runtime::cpu::OpMap dispatcher{
     {TI(ngraph::op::Or), &runtime::cpu::CPU_Emitter::emit<op::Or>},
 };
 
+const size_t runtime::cpu::CPU_ExternalFunction::CPU_ExternalFunction::s_memory_pool_alignment =
+    4096;
+
```

Comment:
why?

Reply:
oh boy. Because `register_pass` uses universal references and takes an address of `s_memory_pool_alignment`.  If  `s_memory_pool_alignment` defined in a class header it won't receive static storage which results in a linking error.